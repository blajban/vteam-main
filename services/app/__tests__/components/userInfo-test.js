import React from 'react';
import renderer from 'react-test-renderer';
import UserInfo from '../../components/userInfo';

jest.mock('expo-constants', () => ({
  manifest: {
    hostUri: 'localhost:3000',
  },
}));
