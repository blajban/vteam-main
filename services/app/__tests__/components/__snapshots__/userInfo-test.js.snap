// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfo Renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "borderColor": "#228377",
      "borderRadius": 10,
      "borderWidth": 5,
      "flex": 1,
      "marginBottom": 30,
      "width": "80%",
    }
  }
/>
`;

exports[`Userinfo Renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "borderColor": "#228377",
      "borderRadius": 10,
      "borderWidth": 5,
      "flex": 1,
      "marginBottom": 30,
      "width": "80%",
    }
  }
/>
`;
